webapp/test/e2e/dialog.test.ts

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+import AppPage from "./pages/AppPage"
```

Comment:
The existing files in this repo DO use semicolon line endings. For consistent style I'd say they need to be added.

Reply:
done - everywhere   
(insert silent weep here)